test/qa/test_firewall.py

```diff
@@ -300,7 +271,12 @@ def test_firewall_lan_discovery(before_connect):
         assert rule not in rules, f"{rule} output rule not found in iptables"
 
 
-def test_firewall_lan_allowlist_interaction():
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_firewall_lan_allowlist_interaction(tech, proto, obfuscated):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
     sh.nordvpn.connect()
```

Comment:
this and the above test have `connect` but no `disconnect`. Shouldn't be a disconnect for them + cleanup firewall?

Reply:
Good point